lacework/resource_lacework_policy_compliance.go

```diff
@@ -0,0 +1,264 @@
+package lacework
+
+import (
+	"errors"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/lacework/go-sdk/api"
+)
+
+func resourceLaceworkPolicyCompliance() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceLaceworkPolicyComplianceCreate,
+		Read:   resourceLaceworkPolicyComplianceRead,
+		Update: resourceLaceworkPolicyComplianceUpdate,
+		Delete: resourceLaceworkPolicyComplianceDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: importLaceworkPolicyCompliance,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"title": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The title of the policy",
+			},
+			"query_id": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The id of the query",
+			},
+			"description": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The description of the query",
+			},
+			"severity": {
+				Type:     schema.TypeString,
+				Required: true,
+				Description: "The severity for the policy. Valid severities are: " +
+					"Critical, High, Medium, Low, Info",
+				StateFunc: func(val interface{}) string {
+					return strings.TrimSpace(strings.ToLower(val.(string)))
+				},
+				ValidateDiagFunc: ValidSeverity(),
+			},
+			"type": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "The policy type must be 'Compliance'",
+				ValidateFunc: func(value interface{}, key string) ([]string, []error) {
+					switch value.(string) {
+					case "Compliance":
+						return nil, nil
+					default:
+						return nil, []error{
+							fmt.Errorf(
+								"%s: can only be 'Compliance'", key,
+							),
+						}
+					}
+				},
+			},
```

Comment:
Since we are adding a new resource that interacts with this Policy type, I don't think we should add this field into the resource schema.
```suggestion
```

Reply:
Instead, just hardcode the type field on all API requests.

Reply:
Removed the type field and hardcoded it to Compliance.